.hashibot.hcl

```diff
@@ -22,3 +22,21 @@ poll "closed_issue_locker" "locker" {
     If you have found a problem that seems similar to this, please open a new issue and complete the issue template so we can capture all the details necessary to investigate further.
   EOF
 }
+
+poll "label_issue_migrater" "remote_plugin_migrater" {
```

Comment:
I'm no native english speaker but I would have wrote that one `migrator` 🤔 ( I'm just being curious )

Reply:
+1 I would also guess `migrator`.

I think this is how is pre-defined in the hashibot.

Reply:
Yup it's just how it is named it seems !